src/main/javascript/.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,14 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\src",
+    "\\src\\pages",
+    "\\src\\pages\\profile",
+    "\\src\\pages\\role",
+    "\\src\\pages\\role\\view",
+    "\\src\\resources\\components",
+    "\\src\\resources\\components\\header"
+  ],
+  "SelectedNode": "\\src\\pages\\role\\view\\view.html",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
These visual studio files are still part of the PR. I'll need you to remove it. You can delete them and then commit it or if you want to keep them, I'm sure there's a way to reset head on these files.